docs/user/checks.rst

```diff
@@ -443,6 +443,23 @@ Ruby format
 
     `Ruby Kernel#sprintf <https://ruby-doc.org/core/Kernel.html#method-i-sprintf>`_
 
+Scheme format
+*************
+
+*Scheme format string does not match source*
+
++------------------------+------------------------------------------------------------+
+| Simple format string   | ``There are ~d apples``                                    |
++------------------------+------------------------------------------------------------+
+| Flag to enable         | `scheme-format`                                            |
++------------------------+------------------------------------------------------------+
+
+.. seealso::
+
+    `Srfi 28 <https://srfi.schemers.org/srfi-28/srfi-28.html>`
+    `Chicken Scheme format <http://wiki.call-cc.org/eggref/5/format>`
+    `Guile Scheme formatted output <https://www.gnu.org/software/guile/manual/html_node/Formatted-Output.html>`
```

Comment:
The links are wrongly formatted (see https://weblate--5601.org.readthedocs.build/en/5601/user/checks.html#scheme-format). Also please use https when possible:

```suggestion
    `Srfi 28 <https://srfi.schemers.org/srfi-28/srfi-28.html>`_,
    `Chicken Scheme format <https://wiki.call-cc.org/eggref/5/format>`_,
    `Guile Scheme formatted output <https://www.gnu.org/software/guile/manual/html_node/Formatted-Output.html>`_
```